### ISSUE TITLE
Feature request : Table style no border

### ISSUE THREAD

**Issue by rblin081** [froala/wysiwyg-editor#1074]
Would it be possible to add a default table style(with corresponding css) that has no borders. The grey borders are helpful when organizing table content, however are not always wanted when displaying outputted html.

**Comment by stefanneculai** [comment]
You could change that in the froala_style CSS file: https://www.froala.com/wysiwyg-editor/docs/examples/editor-style.

**Comment by royshoa** [comment]
Its not a good solutions to edit the CSS, if I like to let my customers add a table with border, and without border, I like them to choose that options, I will be unable to do it if Froala will not let the user choose it and add a related class to it.

**Comment by trstaley** [comment]
I have to agree with royshoa.  This is a styling issue that our clients are requesting.

**Comment by olivierognn** [comment]
Same problem, I Can't add a custom style to tables generated

**Comment by stefanneculai** [comment]
@royshoa @trstaley @olivierognn if you need custom table styles please follow the approach described here: https://www.froala.com/wysiwyg-editor/examples/table-styles.

**Comment by mixlplix** [comment]
Is there going to be a resolution on this?  Having grey borders for every table cell is unacceptable, and why would you do that anyway?  We use tables in invoices that are sent by email so a stylesheet is not going to be helpful.

**Comment by rblin081** [comment]
We were forced to implement our own solution because of inaction on this issue

**Comment by mixlplix** [comment]
Thanks, I just solved it for now by editing the froala-style.css file, removing the borders.

 

</doug>

**Comment by dobon** [comment]
This issue shouldn't have been closed. It was certainly never resolved. 2 years later, the same problem still exists and it annoying my customers. =/

**Comment by Bastek** [comment]
Issue is still present.

**Comment by ikomangmahendra** [comment]
+1 please consider to implement this feature.
For invoice layout we still need to display the border but not for layouting purposes.

**Comment by stefanneculai** [comment]
If you'd like to use a table without any borders, you could define a class for it. E.g.: https://jsfiddle.net/froala/r38jwxbn/408/

<img width="472" alt="screenshot 2018-11-22 at 18 12 41" src="https://user-images.githubusercontent.com/637004/48913818-3ceb1a80-ee82-11e8-84ee-6cdf4324dc7f.png">

**Comment by mayilsaran** [comment]
In angular 8 , how to define a class

**Comment by ffjanhoeck** [comment]
Its still a problem, its sooo sad :( 

I am loading existing HTML into froala and froala adds borders to the table...

**Comment by ArmanGhazaryan94** [comment]
> @stefanneculai The borders inside the editor are really helpful for the customer, because they know where to click. ![image](https://user-images.githubusercontent.com/9037364/127764204-410ad02e-8c9a-43d7-9657-e2cbdd82185e.png)
> 
> But in the end result (the output HTML) I do not want these borders anymore. We are using Froala Editor for writing emails. Because of that we had to set `useClasses` to `false`. So please tell me how I can fix that. ![image](https://user-images.githubusercontent.com/9037364/127764227-85c19d1b-34cc-4b16-a5c2-1712b524df07.png)

any news?

**Comment by dobon** [comment]
@ArmanGhazaryan94 I wrote a custom [tableEditButton](https://froala.com/wysiwyg-editor/docs/options/#tableEditButtons) command for Froala to accomplish borderless table style while using "[useClasses = false](https://froala.com/wysiwyg-editor/docs/options/#useClasses)" feature by toggling `border-width: 0` property on the selected table cells (FYI: setting `border-color` to `transparent` or `border` to `none` didn't work properly due to `useClasses = false` quirks, that's why I used the `border-width` CSS attr specifically). Unfortunately, I can't share that code outright because it is owned by a former employer, but here are the skeletonized steps to follow:

1. Prior to the code you use for configuring and setting up Froala, define and register a [custom button](https://froala.com/wysiwyg-editor/docs/concepts/custom/button/):
```
FroalaEditor.DefineIcon('myToggleTableLinesButton', { NAME: 'star'});

FroalaEditor.RegisterCommand('myToggleTableLinesButton', {
  title: 'Toggle Table Lines',
  focus: false,
  callback: function () {
    this.$el.find('.fr-selected-cell').each(function () {
       $(this).css('border-width', (this.style.borderWidth == '0px') ? '' : '0')
    });
  }
})
```

2. Register the new button using the [tableEditButtons](https://froala.com/wysiwyg-editor/docs/options#tableEditButtons) options. If you were previously using the default tableEditButtons, then remember to include those in the array along with your new custom button.

3. Add Styling to temporarily override the `border-width: 0` styling on the currently selected table cell(s) so that your users don't get too confused about where the table lines are, and they can still be used for drag-to-resize:
```
.fr-element table td.fr-selected-cell, .fr-element table th.fr-selected-cell {
   border: 1px double #1e88e5 !important;
}
```

I hope this helps!
